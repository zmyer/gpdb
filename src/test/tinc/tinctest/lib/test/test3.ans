SELECT 'foo';
 ?column? 
----------
 foo
(1 row)

SELECT 'bar';
 ?column? 
----------
 bar
(1 row)

