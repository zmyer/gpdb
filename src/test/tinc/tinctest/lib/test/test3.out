-- start_ignore

-- end_ignore

-- test test test

-- test
SELECT 'foo';
 ?column? 
----------
 foo
(1 row)

-- another comment

SELECT 'bar';
 ?column? 
----------
 bar
(1 row)

-- another comment here
